model_compression_toolkit/exporter/model_wrapper/keras/builder/node_to_quantizer.py

```diff
@@ -0,0 +1,118 @@
+}
+
+QUANTIZATION_METHOD_2_ACTIVATION_QUANTIZER = {
+    QuantizationMethod.POWER_OF_TWO: keras_inferable_quantizers.ActivationPOTInferableQuantizer,
+    QuantizationMethod.SYMMETRIC: keras_inferable_quantizers.ActivationSymmetricInferableQuantizer,
+    QuantizationMethod.UNIFORM: keras_inferable_quantizers.ActivationUniformInferableQuantizer
+}
+
+
+def get_inferable_quantizer_kwargs(node: BaseNode,
+                                   quantization_target: QuantizationTarget):
+    if quantization_target == QuantizationTarget.Weights:
+        node_w_qc = node.final_weights_quantization_cfg
+        quantization_method = node_w_qc.weights_quantization_method
+        assert quantization_method in QUANTIZATION_METHOD_2_WEIGHTS_QUANTIZER, f'{quantization_method} for weights ' \
+                                                                               f'not in supported quantization ' \
+                                                                               f'methods for inferable quantizers'
+        if quantization_method in [QuantizationMethod.POWER_OF_TWO,
+                                   QuantizationMethod.SYMMETRIC]:
+            return {'num_bits': node_w_qc.weights_n_bits,
+                    'threshold': node_w_qc.weights_quantization_params.get(THRESHOLD),
+                    'signed': True,
+                    'per_channel': node_w_qc.weights_per_channel_threshold,
+                    'channel_axis': node_w_qc.weights_channels_axis}
+
+        elif quantization_method in [QuantizationMethod.UNIFORM]:
+            return {'num_bits': node_w_qc.weights_n_bits,
+                    'per_channel': node_w_qc.weights_per_channel_threshold,
+                    'min_range': node_w_qc.weights_quantization_params.get(RANGE_MIN),
+                    'max_range': node_w_qc.weights_quantization_params.get(RANGE_MAX)}
+        else:
+            Logger.critical(f'Not supported quantization method for inferable quantizers.')
+
+    elif quantization_target == QuantizationTarget.Activation:
+        node_qc = node.final_activation_quantization_cfg
+        quantization_method = node_qc.activation_quantization_method
+        assert quantization_method in QUANTIZATION_METHOD_2_ACTIVATION_QUANTIZER, f'{quantization_method} for weights ' \
+                                                                                  f'not in ' \
+                                                                                  f'supported quantization methods ' \
+                                                                                  f'for inferable' \
+                                                                                  f' quantizers'
+        if quantization_method in [QuantizationMethod.POWER_OF_TWO,
+                                   QuantizationMethod.SYMMETRIC]:
+            return {'num_bits': node_qc.activation_n_bits,
+                    'threshold': node_qc.activation_quantization_params.get(THRESHOLD),
```

Comment:
What about PO2 bool flag? We don't want the convertor to do some calculations on threshold to check it? same for weights quantizers.